airflow/www/static/js/gantt.js

```diff
@@ -212,12 +229,23 @@ d3.gantt = () => {
           mapIndex: d.map_index,
         });
       })
-      .attr('class', (d) => d.state || 'null')
+      .attr('class', (d) => `${d.state || 'null'} all-tasks`)
       .attr('y', 0)
       .attr('transform', rectTransform)
       .attr('height', () => y.rangeBand())
       .attr('width', (d) => d3.max([x(d.end_date.valueOf()) - x(d.start_date.valueOf()), 1]));
 
+    svg.selectAll('.chart')
```

Comment:
Could we add a comment above this svg and the other one to explain that its painting the instance run and the instance queued?

Reply:
sounds good, will add comment here to mark the line painting all instances vs. instance queued states. also will add similar comments in redraw method.

Reply:
Just need to rebase too, to fix the CI issue. And I think this is looking good.